src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -288,12 +292,14 @@ public PaymentModificationResponse execute(@Nullable final Long transactionAmoun
 
     @Override
     public HostedPaymentPageFormDescriptor buildFormDescriptor(final UUID kbAccountId, final Iterable<PluginProperty> customFields, final Iterable<PluginProperty> properties, final CallContext context) throws PaymentPluginApiException {
+        killbillMetrics.markMeter("hpp");
         final Iterable<PluginProperty> mergedProperties = PluginProperties.merge(customFields, properties);
 
         final String amountString = PluginProperties.findPluginPropertyValue(PROPERTY_AMOUNT, mergedProperties);
         Preconditions.checkState(!Strings.isNullOrEmpty(amountString), "amount not specified");
         final BigDecimal amountBD = new BigDecimal(amountString);
         final long amount = amountBD.longValue();
+        killbillMetrics.recordHistogramValue("hpp.amount", amount);
```

Comment:
I think we also only care about ops metrics from the plugin (things like amount is already available from Kill Bill, if we want to build business metrics). I would just *not* send those info.